### PR TITLE
Update SC controls and add JIRA links

### PR DESCRIPTION
**Pull request by redhatrises**

### CODE REVIEW

File: coreos-4/policies/SC-Systems_and_Communications_Protection/component.yaml [hunk 1]
```diff
@@ -8,4 +8,1548 @@
...
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: planned
+  narrative:
+    - text: |
+        The customer is responsible for deployment utilizing namespaces,
+        host roles and other features as well as policies and procedures
+        to implement and ensure continued isolation of security functions
+        from non-security functions.
+        A complete control response is planned. Engineering progress can be
```
Comment (shawndwells) [inline coreos-4/policies/SC-Systems_and_Communications_Protection/component.yaml]: Few occurrences like this. Should be another new line added.

File: openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml [hunk 2]
```diff
@@ -158,317 +275,1305 @@
...
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: planned
+  narrative:
+    - key: a
+      text: |
+        OpenShift has a built-in DNS so that services can be reached by
+        the service DNS as well as the service IP/port. OpenShift supports
+        split DNS by running SkyDNS on the master that answers DNS queries
+        for services.
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml]: The SkyDNS references are from OCP3. We shouldn't carry them forward.

Comment (shawndwells) [reply openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml]: Well, the only references to SkyDNS appear to be OCP 3. For example:

Comment (redhatrises) [reply openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml]: Done

File: coreos-4/policies/SC-Systems_and_Communications_Protection/component.yaml (hunk 1 above)
Comment (redhatrises) [reply coreos-4/policies/SC-Systems_and_Communications_Protection/component.yaml]: These are fixed... not sure why they are not showing the updated fixes.

**Pull request merged**